lightning/src/ln/channel.rs

```diff
@@ -1866,26 +1894,31 @@ impl<ChanSigner: ChannelKeys> Channel<ChanSigner> {
 				// If we were going to send a commitment_signed after the RAA, go ahead and do all
 				// the corresponding HTLC status updates so that get_last_commitment_update
 				// includes the right HTLCs.
-				// Note that this generates a monitor update that we ignore! This is OK since we
-				// won't actually send the commitment_signed that generated the update to the other
-				// side until the latest monitor has been pulled from us and stored.
```

Comment:
14d5b97

Just to be sure, here putting the ChannelMonitorStep storage burden on commitment_signed caller means you don't have anymore to pull latest monitor from Channel at ChannelMonitor restoration. Should be good.

Reply:
Yep. In another PR or two we won't even be able to generate ChannelMonitors from Channels as they won't have a full local copy!